back-end/photo_app/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+from stagram import models
+
+# Create your models here.
+
+
+class Photo(models.Model):
+    caption = models.TextField()
+    user = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name='photos')
+    url = models.URLField()
+
+    def __str__(self):
+        return f'{self.caption}'
```

Comment:
- Change `__str__` to be URL
- Make `caption` nullable
- Don't import `User` from `stgram`, use the built-in Django model
- This morning was stressful with two projects but it's fun just focusing on Slothstagram